market.js

```diff
@@ -129,6 +129,10 @@ class Market extends EventEmitter {
   }
 
   buy (seller, opts) {
+    if (!Buffer.isBuffer(seller)) {
+      seller = Buffer.from(seller.id, 'hex')
+    }
```

Comment:
can we add a comment about how this is to support passing in a dazaar card?

Reply:
done!